src/components/Flex/index.jsx

```diff
@@ -0,0 +1,20 @@
+import * as S from "./Flex.style"
+
+export default function Flex({
+  children,
+  justifyContent = "center",
```

Comment:
I don't think you need default values for justifyContent, alignItems or direction. if we don't pass them we should use CSS default's values